horizons/world/player.py

```diff
@@ -38,7 +38,16 @@
 
 
 class Player(ComponentHolder, WorldObject):
-	"""Class representing a player"""
+	"""Class representing a player
+
+	@type session: horizons.session.Session
+	@param session: Session instance
+	@param worldid: player's worldid
+	@param name: user-chosen name
+	@param color: color of player (as Color)
+	@param clientid: id of client
+	@param inventory: {res: value} that are put in the players inventory
+	"""
```

Comment:
This are the class description and so are the right place above class and not above __init__, so I lern. And in this project are the class description above class, too. And no comment above __init__.  So I make this consistent.
By the way, Epydoc is outdated, a alternative is pdoc https://github.com/BurntSushi/pdoc or Sphinx.